### DISCUSSION TITLE
Keyboard: Rainbow on Zephyrus M16

### DISCUSSION THREAD

**Discussion by Sontarr** [seerge/g-helper#903] [General]
Hi and thank you for your great tool!

Since one of the last versions I can see "Regenbogen" (rainbow) as choice for the keyboard. It doesn't change anything and I believe it should't even be possible because the keyboard of the M16 can only show one colour at a time.

**Comment by seerge** [comment]
@Sontarr hello, on some mono-color devices (like my G14) Rainbow does almost same as Color Cycle (i.e. just smoothly change colors). I can hide it for M16 if it bothers you a lot :)

Btw, does new Heatmap mode works correctly on M16 ?

[GHelper.zip](https://github.com/seerge/g-helper/files/12176888/GHelper.zip)

**Reply by IceStormNG** [reply]
> Btw, does new Heatmap mode works correctly on M16 ?

On my M16 it works fine. Rainbow does nothing though.

**Reply by Sontarr** [reply]
Heatmap is fantastic, thank you!

**Reply by Sontarr** [reply]
> Rainbow does almost same as Color Cycle

That would be okay, but on my M16 (2023) it doesn't change the previously selected mode. Nothing happens at all, if you select Rainbow.
